component/doc/README.md

```diff
@@ -0,0 +1,346 @@
+#Configuring LinkedIn Authenticator
+
+This page provides instructions on how to configure the LinkedIn 
+authenticator and the WSO2 Identity Server using a sample app to 
+demonstrate authentication.You can find more information in the 
+following sections.
```

Comment:
We can remove this sentence as it seems unnecessary: "You can find more information in the 
following sections."